cmd/create/cluster/cmd.go

```diff
@@ -1145,6 +1164,74 @@ func run(cmd *cobra.Command, _ []string) {
 			}
 		}
 	}
+	oidcEndpointUrl := args.oidcEndpointUrl
+	boundServiceAccountSigningKeySecretArn := args.boundServiceAccountSigningKeySecretArn
+	isByoOidcSet := false
+	if isSTS {
+		// Support byo oidc config
+		if interactive.Enabled() {
```

Comment:
Since these should be hidden for now, can you do the same pattern we've used before for hidden flags?
```
if oidcEndpointUrl != "" && interactive.Enabled()
```
That way only users who are aware of the flag _and_ intend on using it get the interactive prompt.